src/Symfony/Component/HtmlSanitizer/composer.json

```
@@ -0,0 +1,33 @@
+{
+    "name": "symfony/html-sanitizer",
+    "type": "library",
+    "description": "Provides an object-oriented API to sanitize untrusted HTML input for safe insertion into a document’s DOM.",
+    "keywords": ["html", "sanitizer", "purifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Titouan Galopin",
+            "email": "galopintitouan@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.0.2",
+        "ext-dom": "*",
+        "league/uri": "^6.5",
+        "masterminds/html5": "^2.4",
+        "psr/log": "^1.0|^2.0|^3.0",
+        "symfony/string": "^5.0|^6.0"
```

```suggestion
        "symfony/string": "^5.4|^6.0"
```

We've bumped all Symfony dependencies to `^5.4|^6.0` on the 6.0  branch. We should do the same for new components.